pkg/storage/cacher.go

```diff
@@ -772,7 +772,7 @@ func (c *cacheWatcher) process(initEvents []watchCacheEvent, resourceVersion uin
 		}
 		// only send events newer than resourceVersion
 		if event.ResourceVersion > resourceVersion {
-			c.sendWatchCacheEvent(event)
+			c.sendWatchCacheEvent(&event)
```

Comment:
same question here, what happens if a watcher mutates an event (or object in an event)?